src/client/user/actions/index.ts

```diff
@@ -934,6 +925,90 @@ const updateTags =
     })
   }
 
+const setQRCodeGenerationMessage =
+  (status: string, callbacks: string[]) =>
```

Comment:
does the term `callbacks` have any specific meaning? it looks like it's only ever used in the context of `jobItemIds` in the frontend, would that name hopefully be more descriptive?

Reply:
Was thinking about reusing it eventually to allow users to retrigger failed jobs, which is why I left the name pretty generic. What do you think?

Reply:
ahh sounds good to leave it as this then since we'll revisit it eventually!